app/src/main/kotlin/com/vrem/wifianalyzer/wifi/accesspoint/AccessPointDetail.kt

```diff
@@ -168,6 +170,20 @@ class AccessPointDetail {
     private fun setViewWiFiBand(view: View, wiFiSignal: WiFiSignal) =
         view.findViewById<TextView>(R.id.wiFiBand)?.setText(wiFiSignal.wiFiBand.textResource)
 
+
+    private fun setViewFastRoaming(view: View, wiFiSignal: WiFiSignal) {
+        view.findViewById<TextView>(R.id.fastRoaming)?.let {
+            if (wiFiSignal.fastRoaming.isEmpty()) {
+                it.setText(R.string.unsupported_802_11_k_v_r)
```

Comment:
Please only display Device API available information. 
Lines 176 - 180: this information is not necessary.

Reply:
It's necessary, because there are two different situations.

1. Below Android R, SDK doesn't support information elements. Users cannot determine whether AP support k/v/r. Use `require_android_11_802_11_k_v_r`.
2. For Android R, some APs do not support k/v/r. Use `unsupported_802_11_k_v_r`

If the output of them are both EMPTY, the users may feel confused about this.

Reply:
There are only 2 states:
1. Fast Roaming (ie 802.11k, 802.11v, 802.11r)
2 Everything else (ie No Fast Roaming, Wrong Android API, etc)

Reply:
> Everything else (ie No Fast Roaming, Wrong Android API, etc)

I don't agree. A blank will confuse the users.

Reply:
Blank means the phone can not support or does not have any information on Fast Roaming protocols.
Do you expect the user to go and buy a new phone with at-least Android R when they see a message `require_android_11_802_11_k_v_r`?

Reply:
Yes, I expect something like this, but I don't mean the user needs to buy **immediately** . It's just an optional feature for future.

For example, the user may have many android devices. When he sees the message, he can choose a newer device. 

Or, the user may ask the friends around, ask them whether his friends have a newer phone and borrows a new phone from his friends.

I'm not a seller of android device... (joke)

Reply:
> For example, the user may have many android devices. When he sees the message, he can choose a newer device.
> Or, the user may ask the friends around, ask them whether his friends have a newer phone and borrows a new phone from his friends.
 
Based on the above, it is definitely that this information is not necessary.